doc/codi.txt

```diff
@@ -264,6 +265,15 @@ g:codi#sync
 
              Default value is 0.
 
+                                                       *g:codi#use_buffer_dir*
+g:codi#use_buffer_dir
+             Whether or not switch to switch to the directory of the buffer
```

Comment:
Yes that is better. Actually now it works differently in Neovim. Perhaps it should explain what and not how...